frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -391,7 +391,7 @@ const ExistingKMSDropDown: React.FC<ExistingKMSDropDownProps> = ({
               );
             }
           } catch (err) {
-            return err;
+            // do nothing
```

Comment:
Maybe just console.error(err) ?

Reply:
the pre commit linting fails, when adding `console.error()`. should we show failure of parsing in the UI?

Reply:
use `// eslint-disable-next-line no-console`